cli/BUILD

```diff
@@ -51,11 +50,10 @@ container_run_and_commit(
     ],
 )
 
-load("@io_bazel_rules_docker//container:container.bzl", "container_image", "container_push")
-
 container_image(
     name = "image_with_dataform_entrypoint",
     base = ":image_with_dataform_cli_commit.tar",
+    cmd = ["help"],
```

Comment:
this would probably be more future-proof if it was just `""`, that way it will always do the default CLI command

(yes it has the same result, but if we change the default we'll only need to do it in one place)

Reply:
I tried this with `[""]` which didn't work as expected, but `[]` does seem to work! Updated.